README.md

```diff
@@ -2,42 +2,160 @@
+
+### The TTX file format
+
+The following tables are currently supported:
+
+    BASE, CBDT, CBLC, CFF, COLR, CPAL, DSIG, EBDT, EBLC, FFTM, GDEF, 
+    GMAP, GPKG, GPOS, GSUB, JSTF, LTSH, MATH, META, OS/2, SING, SVG, 
+    TSI0, TSI1, TSI2, TSI3, TSI5, TSIB, TSID, TSIJ, TSIP, TSIS, TSIV, 
+    VDMX, VORG, avar, cmap, cvt, feat, fpgm, fvar, gasp, glyf, gvar, 
+    hdmx, head, hhea, hmtx, kern, loca, ltag, maxp, meta, name, post, 
+    prep, sbix, vhea, vmtx
+
+Other tables are dumped as hexadecimal data.
+
+TrueType fonts use glyph indices (GlyphIDs) to refer to glyphs in most places.
+While this is fine in binary form, it is really hard to work with for humans. 
+Therefore we use names instead.
+
+The glyph names are either extracted from the `CFF ` table or the `post` table, or are derived from a Unicode `cmap` table. 
+In the latter case the Adobe Glyph List is used to calculate names based on Unicode values. 
+If all of these methods fail, names are invented based on GlyphID (eg `glyph00142`)
+
+It is possible that different glyphs use the same name. 
+If this happens, we force the names to be unique by appending `#n` to the name (`n` being an integer number.)
+The original names are being kept, so this has no influence on a "round tripped" font.
+
+Because the order in which glyphs are stored inside the binary font is important, we maintain an ordered list of glyph names in the font.
+
+### Development and feedback
+
+TTX/FontTools development is ongoing, with an active community of developers including professional developers employed at major software corporations and type foundries as well as hobbyists. 
+
+Feature requests and bug reports are always welcome at <https://github.com/behdad/fonttools/issues/>
+
+The best place for discussions about TTX from an end-user perspective as well as TTX/FontTools development is the <https://groups.google.com/d/forum/fonttools> mailing list.
+You can also email Behdad privately at <behdad@behdad.org>
+
+### Credits
+
+Windows setup script: Adam Twardoch
+
+Icon: Hannes Famira
+
+### Acknowledgements
+
+In alphabetical order:
+
+Olivier Berten,
+Samyak Bhuta,
+Erik van Blokland, 
+Petr van Blokland, 
+Jelle Bosma, 
+Sascha Brawer,
+Tom Byrer,
+Frédéric Coiffier,
+Vincent Connare, 
+Dave Crossland,
+Simon Daniels, 
+Behdad Esfahbod,
+Behnam Esfahbod,
+Hannes Famira, 
+Sam Fishman,
+Matt Fontaine,
+Yannis Haralambous, 
+Greg Hitchcock, 
+Jeremie Hornus,
+Khaled Hosny,
+John Hudson,
+Denis Jacquerye,
+Jack Jansen, 
+Tom Kacvinsky, 
+Jens Kutilek,
+Antoine Leca, 
+Werner Lemberg, 
+Tal Leming,
+Peter Lofting, 
+Cosimo Lupo,
+Project Mashabow,
+Dave Opstad, 
+Laurence Penney, 
+Roozbeh Pournader,
+Garret Rieger,
+Read Roberts, 
+Guido van Rossum, 
+Just van Rossum, 
+Andreas Seidel, 
+Georg Seifert,
+Miguel Sousa,
+Adam Twardoch,
+Adrien Tétar,
+Vitaly Volkov,
+Paul Wise.
+
+### Copyrights
+
+Copyright (c) 1999-2004 Just van Rossum, LettError (just@letterror.com)  
```

Comment:
This git history goes back to 1999 - https://github.com/behdad/fonttools/commits/master?page=57 - so its possible to find all the copyright holders with git log grepping; that's how I produced the above list of people acknowledged. 

Probably we should set AUTHORS and CONTRIBUTORS files and change the copyright notice to `Copyright (c) 1999 - 2015, fontTools Project Authors`